bayesian/auth.py

```diff
@@ -33,16 +39,25 @@ def wrapper(*args, **kwargs):
             header_data = HeaderData(uuid=request.headers.get('uuid', None))
             if header_data.uuid:
                 g.uuid = str(header_data.uuid)
-                user = get_user(g.uuid)
-                if user:
-                    g.user_status = UserStatus[user.status]
+                if ENABLE_USER_CACHING:
+                    logger.debug("Getting user details from cache.")
+                    user = get_user_from_cache(g.uuid)
+                    if user:
```

Comment:
Yes I already tried that, when we get data from cache it is a dictionary object and we can get value using dict["key"], but when we read data from RDS it is a SQL resultSet object and we get a value using row.key. That is why I had to keep it in its respective block, in case of cache it throws error saying dict object does not have attribute status